packages/office-ui-fabric-react/src/components/FloatingPicker/BaseFloatingPicker.types.ts

```diff
@@ -84,14 +83,21 @@ export interface IBaseFloatingPickerProps<T> extends React.ClassAttributes<any>
    * ClassName for the picker.
    */
   className?: string;
+  /**
+   * How to render the Suggestions control used by this floating picker.
+   */
+  onRenderSuggestionControl?: React.ComponentType<BaseFloatingPickerSuggestionProps<T>>;
   /**
    * The properties that will get passed to the Suggestions component.
+   *
+   * @deprecated prefer binding properties on onRendersuggestionControl.
    */
-  pickerSuggestionsProps?: IBaseFloatingPickerSuggestionProps;
+  pickerSuggestionsProps?: IBaseFloatingPickerSuggestionProps<T>;
```

Comment:
Will this now require users to provide a type for `pickerSuggestionProps`? I'm wondering if `T = any` should be here too.